website/docs/reference/supported_commands.md

```diff
@@ -130,3 +130,189 @@ db.collection.aggregate()
+| `$radiansToDegrees`               | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1465) |
+| `$rand`                           | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/541)  |
+| `$range`                          | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1454) |
+| `$rank`                           | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1468) |
+| `$reduce`                         | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1454) |
+| `$regexFind`                      | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1463) |
+| `$regexFindAll`                   | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1463) |
+| `$regexMatch`                     | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1463) |
+| `$replaceOne`                     | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1463) |
+| `$replaceAll`                     | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1463) |
+| `$reverseArray`                   | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1454) |
+| `$round`                          | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1453) |
+| `$rtrim`                          | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1463) |
+| `$sampleRate`                     | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1472) |
+| `$second`                         | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1460) |
+| `$setDifference`                  | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1462) |
+| `$setEquals`                      | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1462) |
+| `$setField`                       | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1461) |
+| `$setIntersection`                | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1462) |
+| `$setIsSubset`                    | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1462) |
+| `$setUnion`                       | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1462) |
+| `$shift`                          | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1468) |
+| `$size`                           | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1454) |
+| `$sin`                            | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1465) |
+| `$sinh`                           | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1465) |
+| `$slice`                          | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1454) |
+| `$sortArray`                      | ⚠      | [Issue](https://github.com/FerretDB/FerretDB/issues/1454) |
```

Comment:
Missing in the check list of https://github.com/FerretDB/FerretDB/issues/1454

Reply:
Fixed! Thanks for noticing :) (somehow, MongoDB didn't mention it in some parts of their docs, probably because it's a relatively new command...)

Reply:
Thanks for creating all of this!